l3kernel/l3keys.dtx

```diff
@@ -3243,6 +3298,12 @@
 %
 % \begin{macro}{\@@_value_or_default:n}
 % \begin{macro}{\@@_default_inherit:}
+% \begin{macro}
+%   {
+%     \@@_value_set:Nn, \@@_value_set:No, \@@_value_set:NV,
+%     \@@_value_set:Nv, \@@_value_set:Ne,
+%   }
+% \begin{macro}{\@@_value_set:NN, \@@_value_set:Nc}
 %   If a value is given, return it as |#1|, otherwise send a default if
```

Comment:
"send" ?? 
Maybe  ==>  set
Or     ==>  return

Reply:
It's basically `\tl_set:Nn`, hence the naming

Reply:
Chris talks about line 3307

Reply:
Yeah, but this is still an indirection for `\tl_set:Nn`, and it's still about what the key value is set to.